.gometalinter.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Deadline": "5m",
+    "Enable": [
+        "errcheck",
+        "gofmt",
```

Comment:
May want to also include `goimports`? Not sure, I think it covers slightly different things from `gofmt`, and goes a little further, but I don't know if it does the simplifications as well.

Reply:
No golint?

Reply:
FWIW here is the config I was playing with on archive: https://github.com/terraform-providers/terraform-provider-archive/compare/f/makelint#diff-df70a2d0f734c21a1ebe861e6e198c2fR25

You can see some of the excludes I added, like ignoring missing errcheck in `defer` calls, etc.

Reply:
I'm not including the others because they are broken at the moment and will be added later: https://github.com/terraform-providers/terraform-provider-aws/pull/5641#issuecomment-415016000

`golint` especially is a nightmare since it wants to massively fix capitalization (e.g. `Id` -> `ID`) and remove underscores for acceptance tests for a bunch of names.